blog/_posts/2018-07-09-welcome-to-team-popcorn.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Meet Team Popcorn
+layout: post
+created_at: Mon July 09 2018
+permalink: blog/2018-07-09-welcome-to-team-popcorn.md
+current: blog
+author: Arati and Jessica
+twitter: TeamPopcornBer
+categories: student-posts
```

Comment:
Just to maintain consistency across files and format for the posts, can you just change how you have declared categories parameter like below:

```
categories:
- student-posts
```